### ISSUE TITLE
1.2.2.3 Reference - on behalf of Josh Jetto

### ISSUE THREAD

**Issue by zctinker** [gryphonshafer/Quizzing-Rule-Book#106]
Current Rule: 
“Reference question type group questions are used to distinguish between exact duplicate words or phrases from the material. The entire reference question is part of the required question and answer.”

Proposed Rule:
Reference question type group questions are used to distinguish between exact duplicate words or phrases from the material. The entire reference question is part of the required question and answer. If the quizzer jumps prior to the quizmaster beginning to speak any portion of the interrogative or Scripture text portion of the question, they are required to provide both the answer as well as the question in order to be called correct.

Example 1: If the quizmaster says, “According to Matthew chapter 10 verse 23” and the quizzer jumps at this point and says, “Truly I tell you, you will not finish going through the towns of Israel before the Son of Man comes,” the quizmaster would then ask, “What is your question?” and the quizzer would have to provide the correct question, which would be “Truly I tell you what?” in order to be called correct. 
Example 1a: If the quizmaster says, “According to Matthew chapter 10 verse 23, Truly…” and the quizzer jumps at this point and says, “Truly I tell you, you will not finish going through the towns of Israel before the Son of Man comes,” the quizmaster would then call the quizzer correct without asking them to provide the correct question, because the quizmaster had said part of the Scripture text portion of the question before the quizzer jumped.

Further explanation: The reason quizzers are required to provide the question as well as the answer if they jump on only the reference portion of a reference question is that this preserves the integrity of the CVR/CR question types by preventing quizzers from simply jumping on the reference portion of the question and quoting the verse. Requiring quizzers to provide the question when they jump on only the reference portion of a reference question prevents CVRs from functionally becoming Quote questions at higher levels of competition. Also, with CR questions, this prevents quizzers from simply jumping on the chapter number and quickly quoting through a cluster of verses within the 5 verse context (potentially up to 11 verses) and hoping that one of the verses they quoted over contains the answer. 

Rationale:
The reasons I propose striking the current part of the rule that states “The entire reference question is part of the required question and answer” and inserting the proposed language are as follows:
1) We don’t require quizzers to give the question on INT and MA questions where the quizzer has jumped before the quizmaster has completed reading the question. CVR and CR question types are simply INT or MA questions that can be asked from more than one place in the quizzing material for that year. The logic of requiring quizzers to provide the question for CVR and CR questions once the quizmaster has begun speaking the interrogative or Scripture text portion of a reference question is inconsistent and confusing. To simply say, “We can require quizzers to provide the question on a Reference question but not on an INT or MA because they’re different question types” is also not logically consistent, because they are not really different question types. A Reference question is simply an INT or MA with a reference clue attached; it is not the same as the difference between an INT and a SIT question or a SIT and a Quote. 
2) Requiring quizzers to provide the question after the quizmaster has gotten beyond the reference portion of the Reference question and has begun speaking the interrogative or Scripture text portion of the question is confusing for all but the highest level of quizzers and is also confusing for coaches and parents and churches who are new or even several years in to quizzing. Requiring quizzers to provide the question once the quizmaster has begun speaking the interrogative or Scripture text portion of the question has been started discourages quizzers who do not have the full material referenced from jumping on questions that they might otherwise answer correctly, and it also discourages them when they do jump on these questions and answer them correctly but don’t have the experience or the quizzing knowledge (NOT Scripture knowledge) to know what the question might be or how to formulate it, and then they are called incorrect in spite of having known and given the answer to the question that had been begun from the text. To say in these cases (and others) that this rulebook and these rules “only apply to Internationals” is really a misnomer. This is the only rulebook published for C&MA Bible Quizzing. If a new District or church is looking to get started in Bible Quizzing, they are going to look to the “Internationals” rulebook as the rules, and they are going to follow them to the confusion, discouragement, and detriment of themselves and of the quizzers they serve. An “Internationals” rulebook that doesn’t work on the local level for new and inexperienced churches, coaches, and quizzers doesn’t work for quizzing. Additionally, because Districts want their International quizzers to have the best experience at Internationals that they can, there is pressure for them to follow the International rulebook at the District level. Rules like this (aside from being illogical) hurt quizzing at the local church and District level, and consequently, at the International level as well (because they shrink the pool of potential International quizzers).
3) Requiring quizzers to provide the question after the quizmaster has begun speaking the interrogative or Scripture text portion of a Reference question does not help quizzers know the Scriptures better. How does requiring a 5th grade girl (or a 12th grade boy for that matter) to provide the question help her to know the text better when she jumps on “The yeast of…” from Matthew 16:6 and answers “The yeast of the Pharisees and Sadducees” and then is called incorrect because she did not provide the question. The argument might be made, “If they have to give the question as well as the answer, then it forces them to know all the references better, so they can get that one question right.” While for a very small percentage of quizzers that might be motivating, I am concerned that even for the vast majority (95-98% of the current, shrinking quizzing population) they see the time commitment as too great (and that task too uninteresting) to want to be able to do that, and so they don’t bother with reference questions. Thus, this rule practically locks out 95% of quizzers from successfully being able to jump on and contribute points to their teams on these question types and serves as a discouragement to quizzers from studying more, because by this rule, we have disincentivized the studying of the Scripture for the vast majority of quizzers. The top 5% were already going to study hard; this rule doesn’t help them know the Scriptures any better. In fact, it probably hurts them in the sense that it reduces the amount of competition they face at the local and District level, since it keeps lower-caliber quizzers from correctly answering more questions, so in a sense, higher-caliber ones don’t have to study as hard in order to win. For competitive, externally-motivated people this does affect motivation to study.

**Comment by jswingle** [comment]
While I agree that one of the reasons for the "What is your question?" prompt is to preserve the integrity of the CVR question type, and that this is less of an issue when the quizzer jumps during the question of the CVR and not the reference, I still think the rule should remain as is.

1. Since we require the question on a CR no matter where the jump is, it is more consistent and less confusing to the quizzers to also require the same thing of CVRs. Even though References are at root just "INTs or MAs with a reference," quizzers certainly think of References in a different category, and the strategy for answering Reference questions is wildly different. I would prefer consistency between CR and CVR questions vs. consistency between CVR and INT/MA questions.

2. I think this point is more important than the first. Namely, that requiring quizzers to give the question on a Reference actually _does_ serve a purpose beyond just preventing CVRs from becoming Quotes. We are asking the quizzer to identify which phrase the Reference question is based around - in other words, which content is repeated elsewhere in a different verse in this chapter? This is part of the point of Reference questions - to "distinguish between similar material." Part of the pedagogical importance of having Reference questions is this incentive to be able to distinguish similar material, and this is preserved by continuing to require the quizzer to give the question if there's a prejump.

**Comment by jswingle** [comment]
I am closing out issue #107 since it is based around the same argument, even though it concerns a different rule. If @JoshJetto 's proposal is accepted, we will want to change both of these rules in the same pull request, so it should be in the same issue.

Current Rule:
“On a Reference type question, if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, ‘What is your question?’”

Proposed Rule:
“On a Reference type question, if the quizzer has given all the information in both the question and the answer and no part of the interrogative or Scripture text portion has been spoken by the quizmaster (i.e. – only part of or the whole of the reference portion of the question had been read before the quizzer jumped), then the quizmaster will ask, ‘What is your question?’”

**Comment by jswingle** [comment]
Issue #97 is identical as well, so I am closing that one out. Most of the discussion has been in this issue.

**Comment by JoshJetto** [comment]
@jswingle This proposed rule change is meant to cover both CVRs and CRs. Thus, there is no inconsistency or confusion there. The only reason this is not given in the examples is because nobody jumps on CRs until some part of the interrogative or Scripture text portion of the question is spoken by the quizmaster. The requirement of the question when the quizmaster has gotten beyond the reference portion of the question was one of the very bad changes written into the rulebook in the 2017 revision that makes Bible Quizzing more confusing and less accessible to 'outsiders.'

**Comment by jttower** [comment]
I agree with @jswingle on this one because of the purpose of Reference questions.  If you remove the requirement for the question, then the quizzer can just quote the verse.  I understand what @JoshJetto means, but I also think it would be too ambiguous to enforce because there would be a range of when quizzers jump.  What if they get one syllable past the reference?  A sound but not a full syllable?  A mouth movement?  And if they jump after the reference on an interrogative word "According to James chapter 3, what...", that doesn't always give enough information that they would automatically get the correct answer.  The strategy of how to answer a Reference question is different than any other question type, which distinguishes it as a question type, and I think we should maintain that.  Removing the question requirement waters down this question type and makes it so that Quote specialists would then take the Reference questions.

I agree that it can be a difficult question type, and if you don't want to require junior quizzers to get the correct question, that's fine, we've already talked about how a district can choose which rules to enforce at other levels of quizzing.  We don't require our B quizzers to get the question correct in order to be called correct, although we do encourage them to try. In this way we are training them for A quizzing and perhaps eventually for IBQ.  But I don't think the argument "it makes quizzing less accessible" is a reason to change the rules.  The rules of rugby or lacrosse or football may be confusing or inaccessible to 95% of the population, but that doesn't mean that the NFL has to change their rules.  Just like certain sports are not for everyone, competitive levels of quizzing may not be for everyone, and that's okay.  I have 5 kids who are/have been in BQ.  Some are extremely motivated and competitive and some do study hard but aren't competitive.  That's okay, and there can be places for all of them in BQ.

**Comment by lukeshev** [comment]
I think we need to keep the rule as is, given that the rulebook is meant to govern Internationals, and in order to avoid reference questions turning into quote questions.
